node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -5,24 +5,7 @@
     <!-- ACCESSIBILITY HELPERS -->
     <a id="topnavigation-skip" class="sr-only oae-skip-link hide" href="#topnavigation-target" accesskey="__MSG__SKIP_TOP_MENU_KEY__">__MSG__SKIP_TOP_MENU__</a>
     <!-- NAVIGATION -->
-    <div class="row">
```

Comment:
We should be able to bring some of this back. With the discussed changes, we now always have a left section, middle section with logo (on large devices) and right section again